### DISCUSSION TITLE
[NextCloud] E019: HTTP status 404. Failed POST request. Check your server configuration and log.

### DISCUSSION THREAD

**Discussion by VPraharsha03** [floccusaddon/floccus#1673] [Q&A]
Logs are empty, I'm using the https://sam.nl.tab.digital/ instance. Any clue ?

Floccus Version: 5.2.4

I've 2FA enabled on my NextCloud Account

**Comment by bilogic** [comment]
yea same problem here

**Reply by marcelklehr** [reply]
> 5.2.4

Can you try updating to latest (v5.3.3)?

**Reply by bilogic** [reply]
How to? https://chromewebstore.google.com/detail/floccus-bookmarks-sync/fnaicdffflnofjppbagibeoednhnbjhg?hl=en is 5.3.1

**Reply by marcelklehr** [reply]
5.3.3 should be out now
